spec/game_spec.rb

```diff
@@ -1 +1,29 @@
 require 'spec_helper'
+
+RSpec.describe Game do
+  before(:each) do
+    @game = Game.new(
+                    game_id: '9',
+                    season: '12',
+                    away_team_id: '14',
+                    home_team_id: '23',
+                    away_goals: '0',
+                    home_goals: '3'
+                  )
+  end
+
+  describe '#initialize' do
+    it '#exists' do
+      expect(@game).to be_a (Game)
+    end
```

Comment:
I put all of these in on 'it' block, should we talk about standardizing this?  I'm happy to change my way of doing it for solidarity.